pkg/apis/componentconfig/types_pluginargs.go

```diff
@@ -131,3 +133,19 @@ type HighNodeUtilizationArgs struct {
 	Thresholds    api.ResourceThresholds
 	NumberOfNodes int
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// DefaultEvictorArgs holds arguments used to configure DefaultEvictorArgs plugin.
+type DefaultEvictorArgs struct {
+	metav1.TypeMeta
+
+	Nodes                   []*v1.Node
+	EvictLocalStoragePods   bool
+	EvictSystemCriticalPods bool
+	IgnorePvcPods           bool
+	EvictFailedBarePods     bool
+	PriorityThresholds      *int32
```

Comment:
```
PriorityThreshold       *PriorityThreshold
```
with
```
type PriorityThreshold struct {
	Value *int32
	Name  string
}
```
so a user can configure either ways. The plugin code will then do https://github.com/kubernetes-sigs/descheduler/pull/781/files#diff-66db71b98f61a3c06ca8d17cbed99626fc23f8bab081cd71a8baf0867613d4d5R78-R89